scripts/collect.sh

```diff
@@ -35,7 +35,8 @@ for desktop in `find $DESKTOP_DIR $DESKTOP_DIR2 -name '*.desktop' | LC_ALL=C sor
     eval $(grep "^X-SuSE-YaST-AutoInst" $desktop | sed -e 's/X-SuSE-YaST-/X_SuSE_YaST_/' )
```

Comment:
Could we avoid this `eval` somehow? If somebody puts a nasty string into a desktop file it can easily cause a shell command injection:

```shell
X-SuSE-YaST-AutoInstResource=kdump; rm -rf /
```

and we have a big trouble... :scream: